README.md

```diff
@@ -11,9 +11,8 @@
 # CyberArk Secrets Provider for Kubernetes
 
 The CyberArk Secrets Provider for Kubernetes enables you to use secrets stored and managed in the CyberArk Vault 
-using DAP or Conjur and consume them in your Openshift application containers. To do so, the CyberArk Secrets 
-Provider for Kubernetes image runs as an init container and provides the Conjur secrets, required by the pod, 
-from DAP or Conjur.
+using DAP or Conjur and consume them in your Openshift application containers. To do so, the CyberArk Secrets Provider for 
```

Comment:
Should this be 
"using DAP or Conjur and consume them in your Kubernetes / Openshift application containers"? 
@orenbm

Reply:
i would use `Kubernetes` but will go with whatever @InbalZilberman decides.

Reply:
Let's have both. Kubernetes / Openshift application